components/UserList.js

```diff
@@ -1,21 +1,107 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Link from 'next/link';
+import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import AccountBoxIcon from '@material-ui/icons/AccountBox';
+import Badge from '@material-ui/core/Badge';
+import { blue as BlueColor } from '@material-ui/core/colors';
 
-const UserList = (props) => {
-  const { users } = props;
+const styless = {
+  root: {
+    flexGrow: 1,
+  },
+  'list-expand': {
+    border: '1px solid #0288D1',
+  },
+  'userFullName-before': {
+    'font-size': '0.875rem',
+    color: '#2196F3',
+  },
+  'userFullName-after': {
+    'font-size': '1.2em',
+    color: '#212121',
+  },
+};
+const bgColor = {
+  start: Object.keys(BlueColor).indexOf('200'),
+  end: Object.keys(BlueColor).indexOf('0'),
+};
+class UserList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.listExpand = this.listExpand.bind(this);
+  }
+  static getFullName(claim) {
+    return `${claim.firstName} ${claim.lastName}`;
+  }
+listExpand = panelIndex => (event, expanded) => {
+  const parent = event.currentTarget.parentElement;
+  let { start } = bgColor;
+  const { end } = bgColor;
+  let operator = 0;
+
+  if (start - end < 0) {
+    operator = end - start;
+    start = end;
+  } else {
+    operator = start - end;
+  }
```

Comment:
This `listExpand` function can be reduced, for example: [link] (https://github.com/ender503/TangleID/blob/2465ed47c548850cfbefd677006513636077d7c2/components/UserList.js#L52)

Reply:
Thanks @ender503  for your demo ，I will try to make more changes。